### DISCUSSION TITLE
UI - Font Size

### DISCUSSION THREAD

**Discussion by lbushey** [TabularEditor/TabularEditor3#515] [🙋♀️TE3 Q&A] [answered]
Hi, I searched documentation and wasn't able to find a way to bump up the font size. My eye sight is pretty poor, any way to make font a bit larger? 

Thanks!

**Comment by otykier** [comment, marked as answer]
This is actually possible by editing the `TabularEditor3.exe.config` file located in the installation folder. We haven't documented it yet, since there's an issue currently, where the .config file gets replaced upon version upgrades.

In the .config file, locate this setting and change the `9` to whatever value you prefer:

```xml
      <setting name="DefaultAppFont" serializeAs="String">
        <value>Segoe UI;9</value>
      </setting>
```

**Reply by lbushey** [reply]
Thank you!

**Reply by lbushey** [reply]
> This is actually possible by editing the `TabularEditor3.exe.config` file located in the installation folder. We haven't documented it yet, since there's an issue currently, where the .config file gets replaced upon version upgrades.
> 
> In the .config file, locate this setting and change the `9` to whatever value you prefer:
> 
> ```
>       <setting name="DefaultAppFont" serializeAs="String">
>         <value>Segoe UI;9</value>
>       </setting>
> ```

Hey! I tried this suggestion and it definitely increased the font size of the surrounding view panes, but font size in Expression Editor is still quite small. Anything else I could try? 
![image](https://user-images.githubusercontent.com/78571501/164083660-da6f082e-4303-4259-b21a-ed3abe535de5.png)

**Reply by otykier** [reply]
All text editors can be zoomed, by holding down CTRL and then scrolling the mouse wheel - but we should probably make them use a default zoom level that corresponds to any font size changes applied in the .config file. We’ll consider that for a future update.

**Comment by joblini** [comment]
Is it possible to change font size in TabularEditor2?  I was unable to locate the DefaultAppFont setting in the TabularEditor.exe.config file.
